nbs/llms.txt

```diff
@@ -0,0 +1,50 @@
+# nbdev
+
+nbdev makes exploration an integral part of your workflow, all while promoting software engineering best practices. It allows you to write, test, document, and distribute software packages and technical articles — all in one place, your notebook.
+
+Here are some of nbdev's features:
+
+- **Supports literate programming**: write prose, code, and tests in notebooks — no context-switching
+- **Git-friendly notebooks**: human-readable merge conflicts; no unwanted metadata
+- **Two-way sync between notebooks and plaintext source code**: allows you to use your IDE for code navigation or quick edits
+- **Beautiful technical documentation**: documentation is automatically generated using Quarto and hosted on GitHub Pages; docs support LaTeX, are searchable, and are automatically hyperlinked (including out-of-the-box support for many packages via nbdev-index)
+- **Seamless testing**: write tests as ordinary notebook cells, run them in parallel with a single command; out-of-the-box continuous integration with GitHub Actions to ensure tests pass before every merge
+- **Simplified package release**: publish packages to PyPI and conda; Python best practices are automatically followed (e.g. only exported objects are included in `__all__`)
+
+## Tutorials
+
+- [Getting started](https://nbdev.fast.ai/getting_started.html.md)
+- [Tutorial](https://nbdev.fast.ai/tutorials/tutorial.html.md)
+- [Best practices](https://nbdev.fast.ai/tutorials/best_practices.html.md)
+
+## API
+
+- [API List](https://nbdev.fast.ai//apilist.txt): A succint list of all functions and methods in nbdev.
```

Comment:
I believe this url is incorrect?  Has an extra slash in the path?

Reply:
fixed